src/core/acroform/PDFAcroField.ts

```diff
@@ -96,6 +110,24 @@ class PDFAcroField {
     return DA?.asString();
   }
 
+  setFontSize(fontSize: number) {
```

Comment:
Please add wrapper methods to [`PDFTextField`](https://github.com/Hopding/pdf-lib/blob/master/src/api/form/PDFTextField.ts), [`PDFButton`](https://github.com/Hopding/pdf-lib/blob/master/src/api/form/PDFButton.ts), [`PDFDropdown`](https://github.com/Hopding/pdf-lib/blob/master/src/api/form/PDFDropdown.ts), and [`PDFOptionList`](https://github.com/Hopding/pdf-lib/blob/master/src/api/form/PDFOptionList.ts) with the appropriate doc comments. Defining the logic in the low-level `PDFAcroField` API makes sense, but this is functionality that should also be available in the high-level API. Most users should not need to access the `.acroField` property or the low-level core APIs.

Note that the wrapper methods should assert that their `fontSize` parameter is non-negative.